source/content/restore-environment-backup.md

```diff
@@ -87,3 +87,37 @@ This is a destructive process. If you're not comfortable with this technique, us
 5. Deploy the change from Dev up to Test and Live.
 
 For more information, see [git reset](https://git-scm.com/docs/git-reset).
+
+## Restoring the LIVE environment
```

Comment:
This section seems to directly conflict with [this warning](https://github.com/pantheon-systems/documentation/pull/4876/files#diff-25d90242c348ba248897ea37c6708856R12) above. Why are we contradicting it?

Reply:
In reality, customers might still need to do this (i.e., They backups of TEST is way behind LIVE, or restoring in TEST will also not work etc.)